demo/src/main/java/com/example/demo/DemoApplicationForSQ.java

```diff
@@ -1,16 +1,14 @@
 package com.example.demo;
 
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-
-@SpringBootApplication
 public class DemoApplicationForSQ {
 
-	public static void main(String[] args) {
-		SpringApplication.run(DemoApplicationForSQ.class, args);
+	public void doSomething() {
 	}
 
-	public void doSomething() {
+	public void doSomething2() {
 	}
 
+	public int doSomething3() {
+		return 777;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method and declare a constant for this value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS3400)